resources/etc/default/openhab2

```diff
@@ -24,6 +24,11 @@ EXTRA_JAVA_OPTS=""
 #OPENHAB_USERDATA=/var/lib/openhab2
 #OPENHAB_LOGDIR=/var/log/openhab2
 #
+# Set the following variable if you'd like a custom backup location.
+# runtime/bin/backup and runtime/bin/restore will use this path for the zip files.
+#
+#OPENHAB_BACKUPS=/var/lib/openhab2/backups
+#
```

Comment:
Wondering if you want to move the EXTRA_JAVA_OPTS section to the bottom. Seems weird, that the file starts with that detail.